fe/fe-core/src/main/java/com/starrocks/connector/delta/ScalarOperationToDeltaLakeExpr.java

```diff
@@ -315,17 +311,24 @@ public Literal visitConstant(ConstantOperator operator, DeltaDataType type) {
                 case DATE:
                     return Literal.ofDate((int) operator.getDate().toLocalDate().toEpochDay());
                 case DATETIME:
+                    LocalDateTime localDateTime = operator.getDatetime();
+                    ZoneId zoneOffset = ZoneOffset.UTC;
                     if (type == DeltaDataType.TIMESTAMP) {
```

Comment:
``` 
if (type == DeltaDataType.TIMESTAMP && !isPartitionColumn)
```

Reply:
The constructor of Literal is different.
Literal.ofTimestamp(value)  
Literal.ofTimestampNtz(value)